packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -371,20 +452,22 @@ export class ApexDebug extends DebugSession {
       projectPath,
       instanceUrl,
       accessToken,
-      clientInfos
+      clientInfos[0],
```

Comment:
Similar with this one. It's not immediately clear what I'm accessing from the clientInfos[0] and clientInfos[1]